application/src/main/java/org/wecancodeit/serverside/controller/CommentController.java

```diff
@@ -1,4 +1,53 @@
 package org.wecancodeit.serverside.controller;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.web.bind.annotation.*;
+import org.wecancodeit.serverside.models.UserName;
+import org.wecancodeit.serverside.repositories.UserNameRepository;
+
+import javax.annotation.Resource;
+import java.util.Collection;
+import java.util.Optional;
+
+@RestController
+@CrossOrigin
 public class CommentController {
+
+    @Resource
+    private CommentRepo commentRepo;
+
+    @GetMaping("/api/comment")
+    public Collection<CommentCollection> getUserNames(){
+        return (Collection<CommentCollection>) userNameRepo.findAll();
+    }
+
+    @PostMapping("/api/users/add-user")
+    public Collection<UserName> addUsers(@RequestBody String body) throws JSONException {
```

Comment:
this entire postMapping doesn't need to exist, don't worry about adding users here, that will be handled in the UserController